explorer/src/components/common/Address.tsx

```diff
@@ -35,6 +37,14 @@ export function Address({
     truncate = true;
   }
 
+  let addressLabel = raw
+    ? address
+    : displayAddress(address, cluster, tokenRegistry);
+
+  if (truncateChars && addressLabel === address) {
+    addressLabel = addressLabel.slice(0, truncateChars) + "...";
```

Comment:
```suggestion
    addressLabel = addressLabel.slice(0, truncateChars) + "…";
```
nit: the ellipsis character takes up less space